calico-cloud/networking/egress/egress-gateway-aws.mdx

```diff
@@ -993,6 +993,15 @@ If you attempt to apply an egress gateway policy that has not been created, {{pr
 
 To add AWS Elastic IPs to the egress gateway pods, follow these steps:
 
+- Ensure that either your worker nodes are in a private subnet, or they are using Elastic IPs for their public IP.
+
+  :::warning
+  If your worker nodes are using ["standard" VPC public IPs](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-ip-addressing.html#vpc-public-ipv4-addresses)
```

Comment:
If your worker nodes are using ["standard" VPC public IPs](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-ip-addressing.html#vpc-public-ipv4-addresses), adding the Elastic IP to the node triggers the node to lose its VPC Public IP. This is because in [AWS networking](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html#eip-basics), a node is not allowed to have a VPC public IP and an Elastic IP. However, we have found this is only enforced at boot time.